src/phoenix/trace/exporter.py

```diff
@@ -33,11 +33,12 @@ def __init__(self) -> None:
         host = get_env_host()
         if host == "0.0.0.0":
             host = "127.0.0.1"
-        endpoint = urljoin(
-            get_env_collector_endpoint() or f"http://{host}:{get_env_port()}",
-            "/v1/traces",
-        )
-        _warn_if_phoenix_is_not_running(endpoint)
+        endpoint = get_env_collector_endpoint()
```

Comment:
It's kinda confusing that we are mixing / matching http and GRPC here? Like up until now people have assumed HTTP

Reply:
If they're using anything other than Phoenix. yes, they would have to switch that over to gRPC when they upgrade Phoenix.